recipes/AISHELL-1/NAR-ASR/NAR_BERT_ASR/laso.py

```diff
@@ -0,0 +1,803 @@
+import torch
```

Comment:
Ok, this is the heart of your work. Here, unfortunately, it is slightly violating one of our rules. Let me explain briefly: if you put your model in recipes/.../laso.py, it won't get installed when peoples are doing pip install speechbrain. In practice, we would need you to move this to speechbrain/models/lobes/laso.py . Before doing that, however, I would need you to do something. I see that you redefine many things that we already provide (all transformer parts). Could you please, for each model part that you redefine, explain us why you had to redefine it instead of using what we already have? The problem is that we can't simply have multiple redefinition of standard models around because it's a nightmare to maintain and not generic. I.e. do you think that your modifications could be build in top of what we already provide for Transformer ? Basically, we have a whole transformer interface that you can just inherit from. I understand that this is quite a bit of work as it would certainly require you to work 1 or 2 full days on your recipe again. But without this step, we unfortunately can't integrate it to SpeechBrain as it would be impossible to maintain properly. Unless there are specific reasons for you to not extend our classes.

Note: I would really like to see your work merged in SB. It looks awesome.